hypha/static_src/src/app/src/containers/ReminderContainer/actions.js

```diff
@@ -0,0 +1,25 @@
+import * as ActionTypes from './constants';
+
+export const initializeAction = (submissionID) => ({
+  type: ActionTypes.INITIALIZE,
+  submissionID
+});
+
+export const deleteReminderAction = (submissionID, reminderID) => ({
+  type: ActionTypes.DELETE_REMINDER,
+  submissionID,
+  reminderID
+});
+
+export const getRemindersSuccessAction = (data) => ({
+  type: ActionTypes.GET_REMINDERS_SUCCESS,
+  data
+});
+
+export const showLoadingAction = () => ({
+	type: ActionTypes.SHOW_LOADING,
+})
+
+export const hideLoadingAction = () => ({
+	type: ActionTypes.HIDE_LOADING,
+})
```

Comment:
Style nitpick:

These two functions have inconsistent indentation. It is indented using tab where as other parts in the part have 2-space indentation.

Same with other occurrences in the changeset.